mkdocs.yml

```diff
@@ -32,3 +32,13 @@ markdown_extensions:
           format: !!python/name:pymdownx.superfences.fence_code_format
   - pymdownx.tabbed:
       alternate_style: true
+extra:
```

Comment:
Do these changes manifest as anything visible on the deployed docs page?

Reply:
Yes, they add some links to the footer.